.github/workflows/build.yaml

```diff
@@ -0,0 +1,27 @@
+name: characters-counter-build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Run checks
+        run: |
+          isort --check --diff . 
+          black --check --diff .
+          flake8 
+          mypy .
+      - name: Run tests with Pytest
+        run: |
+          pytest
```

Comment:
I would love to see the coverage as well. And the part of pytest check to fail the test run if coverage <90% https://pytest-cov.readthedocs.io/en/latest/config.html#reference

Reply:
Done!